framework/auth/core.py

```diff
@@ -147,10 +147,11 @@ def get_user(email=None, password=None, verification_key=None):
 class Auth(object):
 
     def __init__(self, user=None, api_node=None,
-                 private_key=None):
+                 private_key=None, private_link=None):
```

Comment:
Looks like a private key is the `key` attribute on a `privatelink` document in mongo.

![screen shot 2016-01-04 at 12 09 33 pm](https://cloud.githubusercontent.com/assets/134455/12095343/0b3ca1d2-b2dc-11e5-9ad4-d8ed212915c1.png)